library_generation/test/generate_library_integration_test.sh

```diff
@@ -74,23 +87,31 @@ echo "Generating library from ${proto_path}, to ${destination_path}..."
 --grpc_version "${grpc_version}" \
 --transport "${transport}" \
 --rest_numeric_enums "${rest_numeric_enums}" \
+--enable_postprocessing "true" \
+--repo_metadata_json_path "${repo_metadata_json_path}" \
+--owlbot_sha "${owlbot_sha}" \
 --include_samples "${include_samples}"
 
 echo "Generate library finished."
-echo "Checking out googleapis-gen repository..."
-
+echo "Compare generation result..."
 pushd "${output_folder}"
-sparse_clone "${googleapis_gen_url}" "${proto_path}"
 
-echo "Compare generation result..."
 RESULT=0
-diff -r "googleapis-gen/${proto_path}/${destination_path}" "${output_folder}/${destination_path}" -x "*gradle*" || RESULT=$?
-
-if [ ${RESULT} == 0 ] ; then
-  echo "SUCCESS: Comparison finished, no difference is found."
+diff -r "google-cloud-java/${monorepo_folder}" "${destination_path}/workspace" \
```

Comment:
Could you keep the logic of testing against `googleapis-gen` if `enable_postprocessing` is set to `true`?